### PR TITLE
[vllm, lmi-dist] Support input log_probs

### PR DESCRIPTION
**Pull request by sindhuvahinis**
## Description ##

Support input log_probs in our API. 

If user sends `"decoder_input_details": true` and `details: true`, prefill in details is sent. 
If user sends `decoder_input_details:true` and `details: false`, prefill in details is still sent, but other values would be null. 

Testing:
Tested with vllm gpt2 model, with both  json and jsonline output formatter.

### CODE REVIEW

File: engines/python/setup/djl_python/rolling_batch/rolling_batch.py [hunk 1]
```diff
@@ -97,8 +101,11 @@ def _jsonlines_output_formatter(token: Token, first_token: bool,
             final_dict["details"] = {
                 "finish_reason": details.get("finish_reason", None),
                 "generated_tokens": details.get("generated_tokens", None),
-                "inputs": details.get("inputs", None)
+                "inputs": details.get("inputs", None),
             }
+            if "prompt_tokens_details" in details:
```
Comment (lanking520) [inline engines/python/setup/djl_python/rolling_batch/rolling_batch.py:106]: Is this following TGI streaming use case?

Comment (sindhuvahinis) [reply engines/python/setup/djl_python/rolling_batch/rolling_batch.py:106]: Tried TGI container 2.0 

```
data:{"error":"Input validation error: `decoder_input_details` == true is not supported when streaming tokens","error_type":"validation"}
```

Comment (lanking520) [reply engines/python/setup/djl_python/rolling_batch/rolling_batch.py:106]: ok, then we can define the way we like

**Review by lanking520 — APPROVED** [review]

**Review by lanking520 — COMMENTED** [review]
Could we also update our API schema documentation?

**Comment by sindhuvahinis** [comment]
> Could we also update our API schema documentation?

Done

**Pull request merged**